jitterdebugger.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * jitterdebugger - real time response messaurement tool
+ *
+ * Copyright (c) Siemens AG, 2018
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+*/
+
+#ifndef __JITTERDEBUGGER_H
+#define __JITTERDEBUGGER_H
```

Comment:
We don't really need include guards...

Reply:
Sure, though then it doesn't hurt to have them around. If you really dislike them I can drop them for the time being, but be prepare to be screamed at when we need them :)

Reply:
Ok, fair enough, leave them. In another project, I was taught to leave them out whenever possible (to have an better overview of the include chain), but I'm fine with them. :-)

Reply:
I got burned very hard by not having proper  include guards and not being self contained. I am sure there are smart people who are able to figure out all details but I am not one of those and I rather play save :)